plugins/GrapesJsBuilderBundle/Translations/de/javascript.ini

```diff
@@ -8,3 +8,4 @@ grapesjsbuilder.dynamicContentBlockLabel="Dynamischer Inhalt"
 grapesjsbuilder.dynamicContentBtnLabel="Speichern"
 grapesjsbuilder.dynamicContentModalTitle="Dynamischen Inhalt bearbeiten"
 grapesjsbuilder.assetManager.noAssets="Keine <b>Assets</b> hier, zum Hochladen ziehen"
+grapesjsbuilder.buttonBlockLabel="Taste"
```

Comment:
Taste is a correct translation, but not a very common one in this scenario. I checked how it was translated elsewhere. E.g. with the Forms we use the translation "Button" (mautic.form.field.type.button="Button"). The same as in English.